task_DEV-3/Converter.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Text;
+
+namespace task_DEV_3
+{
+    public class Converter
+    {
+        /// <summary>
+        /// The method to convert a number of decimal system to the other.
+        /// </summary>
+        /// <param name="number">Input number.</param>
+        /// <param name="newBase">New base.</param>
+        /// <returns>The number in a new system.</returns>
+        public string Convert(int number, int newBase)
+        {
+            StringBuilder convertedNumberBuilder = new StringBuilder();
```

Comment:
Define allString out of method not to reinitialize each time you call the method